_posts/2017-01-25-docker-volumes.markdown

```diff
@@ -328,7 +328,7 @@ docker container run -v HOST_PATH:CONTAINER_PATH [OPTIONS] IMAGE [CMD]
 Note: HOST_PATH and CONTAINER_PATH can be a folder or file. HOST_PATH must exist before running this command.
```

Comment:
This argument is also incorrect. The `HOST_PATH` doesn't necessarily need to exist before launching the container. Would you mind fixing this as well?